server/settings/loaders.py

```diff
@@ -110,6 +110,9 @@ class ReferenceLoaderModel(BaseSettingsModel):
     display_handle: bool = SettingsField(
         title="Display Handle On Load References"
     )
+    lock_set_on_load: bool = SettingsField(
+        title="Lock Set on Load"
+    )
```

Comment:
This needs to be way more descriptive. Lock set, why? Which set?
Also this only applies to rigs? That **must** be clear then because it's on the ReferenceLoader model.